scenarioexec/util.go

```diff
@@ -42,6 +42,7 @@ func convertAccount(testAcct *mj.Account, world *worldmock.MockWorld) (*worldmoc
 		AsyncCallData:   testAcct.AsyncCallData,
 		ShardID:         uint32(testAcct.Shard.Value),
 		IsSmartContract: len(testAcct.Code.Value) > 0,
+		// CodeMetadata:    testAcct.CodeMetadata.Value,
```

Comment:
TODO: Remove commented Code

Reply:
In the following lines (46-50) we hardcode the metadata to have all permissions by default. This is needed so we don't modify all the tests.